### PR TITLE
[hi] Localize volume.md

### PR DESCRIPTION
**Pull request by i-dipanshu**
Signed-off-by: Dipanshu kumar [medipansh@gmail.com](medipansh@gmail.com)

Localizes content/en/docs/reference/glossary/volume.md to Hindi content/hi/docs/reference/glossary/volume.md

Fixes :- https://github.com/kubernetes/website/issues/35600

### CODE REVIEW

**Comment by i-dipanshu** [comment]
Hey @bishal7679 ,
Could you be please more elaborate what needs to be fixed ?

File: content/hi/docs/reference/glossary/volume.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: वॉल्यूम (Volume)
+id: volume
+date: 2018-04-12
+full_link: /docs/concepts/storage/volumes/
+short_description: >
+  एक डायरेक्टरी जिसमें रहने बाले डेटा एक पोड की कंटेनरों तक अभिगम्य होता हैं।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/volume.md:7]:
```suggestion
  एक डायरेक्टरी जिसमें रहने बाले डेटा एक पॉड की कंटेनरों तक अभिगम्य होता हैं।
```

File: content/hi/docs/reference/glossary/volume.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+aka:
+tags:
+- core-object
+- fundamental
+---
+  एक डायरेक्टरी जिसमें रहने बाले डेटा एक {{< glossary_tooltip text="पोड" term_id="pod" >}} की {{< glossary_tooltip text="कंटेनरों" term_id="container" >}} तक अभिगम्य होता हैं।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/volume.md:14]:
```suggestion
  एक डायरेक्टरी जिसमें रहने बाले डेटा एक {{< glossary_tooltip text="पॉड" term_id="pod" >}} की {{< glossary_tooltip text="कंटेनरों" term_id="container" >}} तक अभिगम्य होता हैं।
```

**Review by Babapool — COMMENTED** [review]
Some spelling nits:

**Comment by bishal7679** [comment]
@i-dipanshu could u pls change your base branch from dev-1.22-hi.1 to main branch

**Pull request closed without merging**